src/TwoFactorAuthenticationPlugin.php

```diff
@@ -47,21 +50,44 @@ public function register(Panel $panel): void
                 ]);
         }
 
+        $panel
+            ->authMiddleware([
+                $this->challengeTwoFactorCheckMiddleware,
+            ]);
+
         if ($this->hasEnforcedTwoFactorSetup()) {
             $panel
                 ->authMiddleware([
-                    EnforceTwoFactorSetup::class,
+                    $this->enforceTwoFactorSetupMiddleware,
                 ]);
         }
     }
 
-    public function enforceTwoFactorSetup(bool $condition = true): static
+    public function setChallengeTwoFactorMiddleware(string $middleware = ChallengeTwoFactor::class): static
+    {
+        $this->challengeTwoFactorCheckMiddleware = $middleware;
+
+        return $this;
+    }
+
+    public function getChallengeTwoFactorMiddleware(): string
+    {
+        return $this->challengeTwoFactorCheckMiddleware;
```

Comment:
Can we rename the `challengeTwoFactorCheckMiddleware` variable to `twoFactorChallengeMiddleware`